server/src-lib/Hasura/Server/Init/Config.hs

```diff
@@ -65,7 +65,8 @@ data RawServeOptions impl
   , rsoEventsHttpPoolSize    :: !(Maybe Int)
   , rsoEventsFetchInterval   :: !(Maybe Milliseconds)
   , rsoLogHeadersFromEnv     :: !Bool
-  , rsoWebSocketCompression :: !Bool
+  , rsoWebSocketCompression  :: !Bool
+  , rsoWebSocketKeepAlive    :: !(Maybe Integer)
```

Comment:
`Int` probably suffices :) 
```suggestion
  , rsoWebSocketKeepAlive    :: !(Maybe Int)
```
(That way we also avoid adding a new `FromENV` instance.)

Reply:
I opted in for `Integer` just because `secondsToDiffTime` accepts it as argument. I could do `fromIntegral` instead.

> Are you doing this as part of Hacktoberfest?

No. I applied for job and wanted to get my feet wet. This is open source project after all, expect PR's :)

Reply:
@jberryman What is the preferred way to construct a `Seconds` instance containing a 5-second Difftime? Using the `Num` instance?